src/main/javacc/CopyTreeParser.java

```diff
@@ -158,6 +154,16 @@ public Object visit(ASTColumnList node, Object data) {
   }
 
   public Object visit(ASTQualifiedName node, Object data) {
+    if(node.jjtGetNumChildren() > 1) {
```

Comment:
can you make this a for loop so it's flexible for any number of children?

Reply:
Done.  This now accounts for multiple namespaces.